compiler-core/src/language_server/tests/action.rs

```diff
@@ -7,8 +7,8 @@ use lsp_types::{
 use super::*;
 
 const TEST_FILE_PATH: &str = match cfg!(target_family = "windows") {
-    true => r"\\?\C:\src\app.gleam",
-    false => "/src/app.gleam",
+    true => r"\\?\C:\src\test_app.gleam",
```

Comment:
Revert this please, having the tests depend on this ordering would make them super brittle.

Reply:
I agree. How do you suggest I fix this problem?

Reply:
I don't have any suggestions here, I've not looked into it I'm afraid.